Microsoft.Toolkit.Uwp.UI.Media/Effects/CrossFadeEffect.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Microsoft.Toolkit.Uwp.UI.Media.Pipelines;
+using Windows.UI.Xaml.Markup;
+
+namespace Microsoft.Toolkit.Uwp.UI.Media.Effects
+{
+    /// <summary>
+    /// A blend effect that merges the current builder with an input one
+    /// </summary>
+    /// <remarks>This effect maps to the Win2D <see cref="Graphics.Canvas.Effects.CrossFadeEffect"/> effect</remarks>
+    [ContentProperty(Name = nameof(Effects))]
+    public sealed class CrossFadeEffect : IPipelineEffect
+    {
+        /// <summary>
+        /// Gets or sets the input to merge with the current instance (defaults to a <see cref="BackdropSourceExtension"/> with <see cref="Windows.UI.Xaml.Media.AcrylicBackgroundSource.Backdrop"/> source).
+        /// </summary>
+        public PipelineBuilder Source { get; set; }
```

Comment:
Was just thinking, is there a way to provide a whole other pipeline in XAML yet?

```xml
<brushes:PipelineBrush Source="{effects:BackdropSource}">
  <effects:LuminanceToAlphaEffect/>
  <effects:CrossFadeEffect>
    <effects:CrossFadeEffect.Source>
       <effects:PipelineSource Source="{effects:ImageSource Uri=ms-appx:///SamplePages/DropShadowPanel/Unicorn.png}">
         <effects:ShadeEffect Color="#FF222222" Intensity="0.2"/>
       </effects:PipelineSource>
    </effects:CrossFadeEffect.Source>
  <efffects:CrossFadeEffect/>
  <effects:BlurEffect Amount="16"/>
</brushes:PipelineBrush>
```

Or is that more of the point of the `Effects` to start a new pipeline at the CrossFade point?

```xml
<brushes:PipelineBrush Source="{effects:BackdropSource}">
  <effects:LuminanceToAlphaEffect/>
  <effects:CrossFadeEffect Source="{effects:ImageSource Uri=ms-appx:///SamplePages/DropShadowPanel/Unicorn.png}">
    <effects:ShadeEffect Color="#FF222222" Intensity="0.2"/>
  <efffects:CrossFadeEffect/>
  <effects:BlurEffect Amount="16"/>
</brushes:PipelineBrush>
```

Reply:
Got it, we're good, thanks!